scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -13,19 +13,36 @@
    "cell_type": "markdown",
```

Comment:
<p>DATA_PATH_TRAIN can be removed here since it is already specified above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>